src/rpc/blockchain.cpp

```diff
@@ -1520,15 +1589,41 @@ static RPCHelpMan getmempoolinfo()
                         {RPCResult::Type::NUM, "maxmempool", "Maximum memory usage for the mempool"},
                         {RPCResult::Type::STR_AMOUNT, "mempoolminfee", "Minimum fee rate in " + CURRENCY_UNIT + "/kB for tx to be accepted. Is the maximum of minrelaytxfee and minimum mempool fee"},
                         {RPCResult::Type::STR_AMOUNT, "minrelaytxfee", "Current minimum relay fee for transactions"},
-                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"}
+                        {RPCResult::Type::NUM, "unbroadcastcount", "Current number of transactions that haven't passed initial broadcast yet"},
+                        {RPCResult::Type::OBJ, "fee_histogram", "",
+                            {
+                                {RPCResult::Type::OBJ, "<feerate-group>", "Object per feerate group",
```

Comment:
Looks like the output doc is missing the "ranges" JSON object returned by the RPC output that emcompasses the feerate groups:

help
```
"fee_histogram" : {        (json object)
    "<feerate-group>" : {    (json object) Object per feerate group
      "sizes" : n,           (numeric) Cumulated size of all transactions in feerate group
      "count" : n,           (numeric) Amount of transactions in feerate group
      "fees" : n,            (numeric) Cumulated fee of all transactions in feerate group
      "from_feerate" : n,    (numeric) Group contains transaction with feerates equal or greater than this value
      "to_feerate" : n       (numeric) Group contains transaction with feerates less than than this value
    },
```
actual output
```
  "fee_histogram": {
    "ranges": {
      "0.00000001": {
        "sizes": 0,
        "count": 0,
        "fees": 0,
        "from_feerate": "0.00000001",
        "to_feerate": "0.0000001"
      },
```

Reply:
I see. Will fix. Thank you

Reply:
It should be fixed now. Even though I still think `ranges` is not a good word. `groups`?

Reply:
maybe `fee rate groups`

Reply:
I have changed it to `fee_rate_groups`. I think it looks nicer now. What do you think?

Reply:
LGTM